### PR TITLE
MISUV-3718 Update Claim a Refund 

### PR DESCRIPTION
**Pull request by AshokGitAccount**
Fix for Accessibility issues and updated tests

### CODE REVIEW

File: app/views/CreditAndRefunds.scala.html [hunk 1]
```diff
@@ -125,6 +134,14 @@ <h2 class="govuk-heading-s">@{balance.get.firstPendingAmountRequested.get.toCurr
                     }
                     @if(!charge._1.documentDetail.validMFACreditDescription()) {
                         <li>
+                            <p class="govuk-heading-s">
+                                @charge._1.documentDetail.paymentOrChargeCredit.get.toCurrencyString
+                                <span class="govuk-body">@messages("credit-and-refund.payment")
+                                    <a href="@getControllerHref(charge._1.documentDetail, isAgent)"
+                                        class="govuk-link">@charge._1.dueDate.get.toLongDate
```
Comment (avelx) [inline app/views/CreditAndRefunds.scala.html:141]: pls avoid calling .get directly, use .getOrElse instead

File: app/views/CreditAndRefunds.scala.html [hunk 2]
```diff
@@ -109,6 +109,15 @@ <h2 class="govuk-heading-s">@{balance.get.firstPendingAmountRequested.get.toCurr
                 @if(isMFACreditsAndDebitsEnabled){
                     @if(charge._1.documentDetail.validMFACreditDescription()) {
                         <li>
+                            <p class="govuk-heading-s">
+                                @charge._1.documentDetail.paymentOrChargeCredit.get.toCurrencyString
```
Comment (avelx) [inline app/views/CreditAndRefunds.scala.html:113]: pls avoid calling .get directly, use .getOrElse instead

**Review by avelx — APPROVED** [review]
minor changes required, otherwise looks good

**Review by avelx — CHANGES_REQUESTED** [review]
minor changes required

File: app/views/CreditAndRefunds.scala.html (hunk 1 above)
Comment (AshokGitAccount) [reply app/views/CreditAndRefunds.scala.html]: Updated .get with .getOrElse

**Review by avelx — APPROVED** [review]
looks good

**Pull request merged**